amulet/api/version_loader.py

```diff
@@ -4,13 +4,13 @@
 import os
 import glob
 import json
-import importlib
+from importlib.util import spec_from_file_location, module_from_spec
```

Comment:
The import here is different than the other usage of this function

Reply:
Yeah, I'll fix the import in `format_loader.py` to match how it's used here